style/sass/README.md

```diff
@@ -27,29 +27,178 @@
 * Use a `%` unit for the amount/weight when using Sass's color functions: `darken($color, 20%)`, not `darken($color, 20)`
 * Use a trailing comma after each item in a map, including the last item.
 
-## Order
-
-* Use alphabetical order for declarations.
-* Place `@extends` and `@includes` at the top of your declaration list.
-* Place media queries directly after the declaration list.
-* Place pseudo-classes and pseudo-elements second.
-* Place nested elements third.
-* Place nested classes fourth.
-
 ## Selectors
 
-* Don't use ID's for style.
-* Avoid over-qualified selectors: `h1.page-title`, `div > .page-title`
 * Use meaningful names: `$visual-grid-color` not `$color` or `$vslgrd-clr`.
-* Be consistent about naming conventions for classes. For instance, if a project is using BEM, continue using it, and if it's not, do not introduce it.
 * Use ID and class names that are as short as possible but as long as necessary.
 * Avoid nesting more than 3 selectors deep.
 * Avoid using comma delimited selectors.
 * Avoid nesting within a media query.
-* Don't concatenate selectors using Sass's parent selector (`&`).
 
 ## Organization
 
 * Use a `base` directory for styling element selectors, global variables, global extends and global mixins.
 * Use HTML structure for ordering of selectors. Don't just put styles at the bottom of the Sass file.
 * Avoid having files longer than 100 lines.
+
+---
+
+## Selectors
+
+### Selector specificity
+
+- Don't use ID selectors.
+- Avoid over-qualified selectors, e.g. `h1.page-title`.
+
+<details>
```

Comment:
Just throwing this out here for discussion: could we name this element something other than "Details?" perhaps something like "More" or "Tell me why" or "The Why" or anything better than those suggestions 😄

Reply:
This makes use of the newer [`<details>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) element. I think GitHub polyfills support.

I think we can specify "More", "Tell me why", or "The Why" in a [`<summary>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/summary) element:

<details>
  <summary>Click on me to expand!</summary>

```html
<details>
  <summary>Click on me to expand!</summary>
</details>
```
</details>

Reply:
I like "Tell me why." You're not clicking if you're using this on mobile!

Reply:
I just reviewed this and tried out some of the suggestions, but I think I like the default “Details”…

“Tell me why” seems to describe the “Motivation” content that is expanded, but we’re also nesting “Resources” “Linting”, and “Code examples” which don’t fit the descriptor “tell me why.” Details, to me, seems to be more inclusive to all the things we’re nesting within the `<details>` element.